yarn-project/circuit-types/src/tx_execution_request.ts

```diff
@@ -140,4 +141,8 @@ export class TxExecutionRequest {
       [AuthWitness.random()],
     );
   }
+
+  [inspect.custom]() {
+    return `TxExecutionRequest(${this.functionSelector}(${this.argsOfCalls.map(arg => arg.toString()).join(', ')}))`;
```

Comment:
Outputting `argsOfCalls` here would be misleading, since these are the args for all nested function calls that the entrypoint method is expected to call. Not just that, but `toString` on `PackedValues` probably won't give you anything meaningful.

What you probably want is to log the `values` of the `argsOfCalls` with a `hash` that equals the `firstCallArgsHash` in the execution request. Or, even easier, just not log out args. I'd vote for the latter, and throw in the `origin` which should allow you to track which account contract this call corresponds to.